frontend/src/i18n/index.ts

```diff
@@ -20,6 +23,13 @@ i18n
   .init({
     fallbackLng: "en",
     debug: process.env.NODE_ENV === "development",
+    backend: {
```

Comment:
If I understand correctly, this is unnecessary based on https://github.com/OpenDevin/OpenDevin/pull/851#pullrequestreview-1985179133

Reply:
> If I understand correctly, this is unnecessary based on [#851 (review)](https://github.com/OpenDevin/OpenDevin/pull/851#pullrequestreview-1985179133)

True thx